packages/core/admin/server/controllers/permission.js

```diff
@@ -44,4 +44,10 @@ module.exports = {
       },
     };
   },
+
+  async getContentApiPermissions(ctx) {
```

Comment:
I don't know if it makes sense to put the content-api related stuff next to the RBAC permissions. Maybe we could create a content-api controller instead. What do you think?

Reply:
that was my intial plan but I thought it's only this one function so I placed it alongside the permissions, but it might be good to separate, because it seems we might want to add more stuff to it than just pemrissions